### ISSUE TITLE
Training Problem

### ISSUE THREAD

**Issue by sosown123** [autonomousvision/occupancy_networks#50]
I download the Preprocessed data and unzip it to the data/ShapeNet folder
I met some problems in the following messages, could you help me?
It seems like that i did not load the data 

File "/media/mickyv2/micky/occupancy_networks-master/train.py", line 70, in <module>
    data_vis = next(iter(vis_loader))
  File "/home/mickyv2/anaconda3/envs/mesh_funcspace/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 615, in __next__
    batch = self.collate_fn([self.dataset[i] for i in indices])
  File "/media/mickyv2/micky/occupancy_networks-master/im2mesh/data/core.py", line 169, in collate_remove_none
    return data.dataloader.default_collate(batch)
  File "/home/mickyv2/anaconda3/envs/mesh_funcspace/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 200, in default_collate
    elem_type = type(batch[0])
IndexError: list index out of range

**Comment by mnunezu** [comment]
Up!, did you solve it?

**Comment by mnunezu** [comment]
I solve it, you have to specify the img_folder corresponding in the dataset.

in the case of core dataset img_folder: img_choy2016

**Comment by NaveedMaz** [comment]
@mnunezu can you please specify which line and where i am noob in this. so please help

**Comment by tiexuedanxin** [comment]
> @mnunezu can you please specify which line and where i am noob in this. so please help

hello, Have you solve the problem?

**Comment by NaveedMaz** [comment]
yes i solved that issue but the problem is that i managed to get the chamfer loss 0.0016 and that is very low on pixel2mesh implementation inside this repo but i am unable to draw predictions using this trained model. it needs camera configuration points which it should not need during prediction.
Anyone could help me with this?

**Comment by tiexuedanxin** [comment]
> yes i solved that issue but the problem is that i managed to get the chamfer loss 0.0016 and that is very low on pixel2mesh implementation inside this repo but i am unable to draw predictions using this trained model. it needs camera configuration points which it should not need during prediction.
> Anyone could help me with this?

Could you tell me how you solve it?

**Comment by NaveedMaz** [comment]
yes, sure you have to change the img dir in your configuration file. such as img_folder:  img_choy2016

**Comment by NaveedMaz** [comment]
i hope it helps otherwise i can send you the file or email you if you like.
just a quick suggestion rather wasting time on it you can always to and check out pytorch3d

**Comment by tiexuedanxin** [comment]
> i hope it helps otherwise i can send you the file or email you if you like.
> just a quick suggestion rather wasting time on it you can always to and check out pytorch3d

hello, my eamil address is 740337741@qq.com, very thank you for send me the file.

**Comment by tiexuedanxin** [comment]
> i hope it helps otherwise i can send you the file or email you if you like.
> just a quick suggestion rather wasting time on it you can always to and check out pytorch3d

hello, please help.

**Comment by tiexuedanxin** [comment]
> i hope it helps otherwise i can send you the file or email you if you like.
> just a quick suggestion rather wasting time on it you can always to and check out pytorch3d

hello,
